packages/stitch/src/createDelegationPlanBuilder.ts

```diff
@@ -183,15 +183,15 @@ function calculateDelegationStage(
   };
 }
 
-function calculateScore(selections: readonly SelectionNode[] | SelectionNode[]) {
+export function calculateSelectionScore(selections: readonly SelectionNode[] | SelectionNode[]) {
```

Comment:
I don't think the mutable type is necessary.
```suggestion
export function calculateSelectionScore(selections: readonly SelectionNode[]) {
```

Reply:
Done!